src/main.rs

```diff
@@ -235,7 +235,7 @@ fn handle_command(message: &Message, text: &str) -> ReplyMsg {
         Ok(command) => {
             let msg_username = match message.from.username.as_ref() {
                 Some(msg_username) => msg_username.to_owned(),
-                None => return format_error("To play this game you need a username"),
+                None => return format_error("You must register an username in your Telegram in order to use this bot. Set a usernameame in yoour telegram app settings."),
```

Comment:
There is an extra `o` on `your` 😉

Reply:
ans 'usernameame' hahaha gonna fix both thanks

Reply:
Nossa, não vi kkk